src/simulation/simulation_transient_voltage_graph.ts

```diff
@@ -1,18 +1,19 @@
 import { z } from "zod"
 import { getZodPrefixedIdWithDefault } from "src/common"
+import { duration, timestamp } from "src/units"
 import { expectTypesMatch } from "src/utils/expect-types-match"
 
 export interface SimulationTransientVoltageGraph {
   type: "simulation_transient_voltage_graph"
   simulation_transient_voltage_graph_id: string
   simulation_experiment_id: string
-  timestamps_ms?: number[]
+  timestamps?: number[]
   voltage_levels: number[]
   schematic_voltage_probe_id?: string
   subcircuit_connecivity_map_key?: string
   time_per_step: number
-  start_time_ms: number
-  end_time_ms: number
+  start_timestamp: number
```

Comment:
Keep the original, its more clear and i was wrong these arent timestamps, they are “ms.optional()”